CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 
 #### Enhancements
 
+* Support iOSApplicationExtension, macOSApplicationExtension, watchOSApplicationExtension, and tvOSApplicationExtension identifiers in `deployment_target` rule. To configure the rule for these identidiers, you need to use these keys: `iOSApplicationExtension_deployment_target`, `macOSApplicationExtension_deployment_target`, `watchOSApplicationExtension_deployment_target`, `tvOSApplicationExtension_deployment_target`.
```

Comment:
Please add line breaks after about every 80 characters and add two additional spaces at the end of the description. They are required to render the CHANGELOG properly:
```suggestion
* Support `iOSApplicationExtension`, `macOSApplicationExtension`,
  `watchOSApplicationExtension`, and `tvOSApplicationExtension` identifiers
  in the `deployment_target` rule. To configure the rule for these identifiers,
  you need to use the keys `iOSApplicationExtension_deployment_target`,
  `macOSApplicationExtension_deployment_target`,
  `watchOSApplicationExtension_deployment_target`, and
  `tvOSApplicationExtension_deployment_target`.  
```